File: 1-iniciante/1036/1036.lua
A, B, C = io.read("*n","*n","*n")

if A == 0 then
	print("Impossivel calcular")
	return
end

discriminant = B^2 - (4 * A * C)
if discriminant < 0 then
	print("Impossivel calcular")
	return
end

root1 = (-B + math.sqrt(discriminant)) / (2 * A)
root2 = (-B - math.sqrt(discriminant)) / (2 * A)
print("R1 = " .. string.format("%.5f", root1))
print("R2 = " .. string.format("%.5f", root2))

File: 1-iniciante/1036/1036.c
#include <stdio.h>
#include <math.h>

int main() {
    float A, B, C, discriminant, root1, root2;
    scanf("%f%f%f", &A, &B, &C);

    if (A == 0) {
        printf("Impossivel calcular\n");
        return 0;
    }

    discriminant = (B * B) - (4 * A * C);
    if (discriminant < 0) {
        printf("Impossivel calcular\n");
        return 0;
    }

    root1 = (-B + sqrt(discriminant)) / (2 * A);
    root2 = (-B - sqrt(discriminant)) / (2 * A);
    printf("R1 = %.5f\n", root1);
    printf("R2 = %.5f\n", root2);
    return 0;
}